src/hangman.js

```diff
@@ -1,78 +1,100 @@
+  console.clear();
+  console.log(
+    boxen(
+      `Hangman is a classic game, where you need to guess the specific word by one letter.\nFor each mistake - your character will be one step closer to being hanged.\nBefore the game starts, you need to choose a word theme (sports, science, movies, etc).\n\nGood luck, ${username}. Save this poor guy.`,
+      { title: "Hangman", textAlignment: "center", titleAlignment: "center" }
+    )
+  );
 
-    await delay(2000);
-    if (readyChecker()) {
-        gameStarts();
-        preHangman(username);
-    } else {
-        gameEnds();
-    }
+  await delay(2000);
+  if (readyChecker()) {
+    gameStarts();
+    preHangman(username);
+  } else {
+    gameEnds();
+  }
 };
 
 const preHangman = async (username) => {
-    console.clear();
-    const themes = getThemesList();
-    console.log(themes);
-    const userThemeNum = readlineSync.question('Choose your theme: ');
-    isThemeValid(userThemeNum) ? startHangman(username, userThemeNum) : console.log(`Theme with number ${userThemeNum} is not valid.\nReturning to main menu...`);
+  console.clear();
+  const themes = getThemesList();
+  console.log(themes);
+  const userThemeNum = readlineSync.question("Choose your theme: ");
+  isThemeValid(userThemeNum)
+    ? startHangman(username, userThemeNum)
+    : console.log(
+        `Theme with number ${userThemeNum} is not valid.\nReturning to main menu...`
+      );
 
-    await delay(3000);
-    mainMenu(username);
+  await delay(3000);
+  mainMenu(username);
 };
 
 const startHangman = async (username, userThemeNum) => {
-    console.clear();
-    const themeName = getThemeName(userThemeNum);
-    console.log(`Selected theme is - ${themeName}.\nPicking up an appropriate word for you...`);
-    const goalWord = getThemeWord(themeName);
-    let goalWordHidden = hideWord(goalWord);
-    let depressionLevel = 0;
-    let correctAnswers = 0;
+  debugger;
+  console.clear();
+  const themeName = getThemeName(userThemeNum);
+  console.log(
+    `Selected theme is - ${themeName}.\nPicking up an appropriate word for you...`
+  );
+  const goalWord = getThemeWord(themeName);
+  let goalWordHidden = hideWord(goalWord);
+  let depressionLevel = 0;
+  let correctAnswers = 0;
 
-    await delay(500);
-    while (depressionLevel < 5 && correctAnswers < goalWord.length) {
-        console.log(`Current depression level: ${depressionLevel}`);
+  await delay(500);
+  while (depressionLevel < 5 && correctAnswers < goalWord.length) {
+    console.log(`Current depression level: ${depressionLevel}`);
 
-        await delay(1000);
-        console.log(`Theme: ${themeName}\n\n${goalWordHidden}`);
+    // await delay(1000);

```

Comment:
You have a bug in this `await delay()` functions, cuz its returns `Promise<void>`, i think its make it.